api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -13,27 +13,38 @@ A user can click the **Export** button to save an Excel file with the exported d
 
 The following instructions show how to enable and configure client-side export:
 
-1. **Reference the required libraries**  
-    Reference or import the <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1 or newer and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> libraries. Export also requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip</a> library. In a modular environment, this library is listed in package dependencies and is already added. If your app does not use modules, reference JSZip manually.
+1. **Install or reference the required libraries**  
+    You need the following libraries: <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v4.1.1 and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> v2.0.2.
 
     ---
     ##### jQuery
         <!--HTML-->
         <head>
             <!-- ... -->
             <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.4.0/polyfill.min.js"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/exceljs/3.3.1/exceljs.min.js"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/1.3.8/FileSaver.min.js"></script>
-            <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.5/jszip.min.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/exceljs/4.1.1/exceljs.min.js"></script>
+            <script src="https://cdnjs.cloudflare.com/ajax/libs/FileSaver.js/2.0.2/FileSaver.min.js"></script>
             <!-- reference the DevExtreme sources here -->
         </head>
 
     ##### Angular   
 
+        <!-- tab: Installation command -->
```

Comment:
If you remove the tab name, will it be displayed one under another with tsconfig.app.json?

Reply:
I'm not really sure that I got you right, bu different options (<!-- --> or without it at all) produce the following output:
![image](https://user-images.githubusercontent.com/33040934/93361509-9db0c780-f84d-11ea-9958-44196567868d.png)
![image](https://user-images.githubusercontent.com/33040934/93361543-a6090280-f84d-11ea-8276-8bd61c8e160d.png)

Reply:
Okay, Let's keep it as is.